src/group.rs

```diff
@@ -157,22 +162,35 @@ impl GroupOps for crate::IronOxide {
         rt.block_on(group_api::list(self.device.auth(), None))
     }
 
-    fn group_create(&self, opts: &GroupCreateOpts) -> Result<GroupMetaResult> {
+    fn group_create(&self, opts: &GroupCreateOpts) -> Result<GroupCreateResult> {
```

Comment:
We don't usually do any business logic up in this layer of the api so this logic should go down into group_api::group_create instead of being done up here.

Reply:
As we talked about, I'll need some help getting the logic into the other layer